client/src/components/Search/SearchBox.js

```diff
@@ -27,7 +27,7 @@ class SearchBox extends Component {
    * @param {Object} event
    */
   handleKeyUp(event) {
-    if (event.keyCode === 13) {
+    if (event.key === 'Enter') {
```

Comment:
Somehow I find 13 to be more intuitive than "Enter". Does that make me weird? :wink:

Reply:
Yes, yes it does 😜 The main reason I changed it was keyCode is deprecated and it also made it easier to test as I could write a step with and `I press the "Enter" key`